nfs-ganesha/build/build_deb

```diff
@@ -3,14 +3,21 @@ set -ex
 
 # Only do actual work when we are a DEB distro
 if test "$DISTRO" != "debian" -a "$DISTRO" != "ubuntu"; then
-    exit 0
+    exit 1
 fi
 
+REPO_URL="https://shaman.ceph.com/api/repos/ceph/$CEPH_BRANCH/$CEPH_SHA1/$DISTRO/$DIST/repo"
+if `curl --fail -L $REPO_URL`; then
+  echo "Ceph debian lib repo exists in shaman"
+else
+  echo "Ceph debian lib repo does NOT exist in shaman"
+  exit 1
+fi
 
 # We need this for system, to build ganesha, and to enable FSAL_CEPH and 
 # FSAL_RGW in the .spec file when cmake command runs
 sudo apt-get update
-sudo apt-get -y install cmake equivs libkrb5-dev bison flex quilt librgw-dev libcephfs-dev
+sudo apt-get -y install cmake equivs libkrb5-dev bison flex quilt ceph
```

Comment:
strange i was getting errors telling me I should install the full ceph package instead of just those two libraries but I just did a run of those two and it passed.

I'll make that change and send it back over here